eva/udfs/abstract/hf_abstract_udf.py

```diff
@@ -80,11 +80,14 @@ def output_formatter(self, outputs: Any):
             return pd.DataFrame(outputs, index=[0])
         # PERF: Can improve performance by avoiding redundant list creation
         result_list = []
-        for row_output in outputs:
-            # account for the case where we have more than one prediction for an input
-            if isinstance(row_output, list):
-                row_output = {k: [dic[k] for dic in row_output] for k in row_output[0]}
-            result_list.append(row_output)
+        if outputs != [[]]:
```

Comment:
I think this can be made cleaner by returning an empty Dataframe when NER model returns empty list.

Reply:
by returning the empty dataframe it dosen't work it gives unknown columns finds error

Reply:
Yeah but in the sample case, we do have entities so I believe that shouldn't happen?

Reply:
yes, In sample case we have the entities and where this unknown columns issue hasn't been occurred